app/pb_kits/playbook/pb_text_input/_text_input.jsx

```diff
@@ -57,26 +60,26 @@ const TextInput = ({
     >
       <Caption
           className="pb_text_input_kit_label"
-          dark={dark}
           text={label}
       />
-      <div className={classnames('text_input_wrapper', globalProps(props))}>
+      <div className="text_input_wrapper">
         <If condition={children}>
           {children}
           <Else />
           <input
               {...props}
               className="text_input"
+              disabled={disabled}
```

Comment:
I think you can simplify this prop and I think we should add a doc example for disabled state here https://playbook.powerapp.cloud/kits/text_input/react

```suggestion
              disabled
```

Reply:
Without the `={disabled}` the HTML actually just interprets it as a disabled field.